content/uc-doc/contributors/style_guide.md

```diff
@@ -6,363 +6,403 @@ title: Style Guide
 
 ### License {#license}
 
-Python files start with a UTF8 encoding comment and the GPLv3 license. A
-blank line should separate the license from the imports
+Python files the GPLv3 license. A blank line should separate the license from the imports
 
 Example:
 
-    # -*- coding: utf-8 -*-
-    # Copyright 2016 The Wazo Authors  (see the AUTHORS file)
-    # SPDX-License-Identifier: GPL-3.0-or-later
+```python
+# Copyright 2021 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
 
-    import argparse
+import argparse
+```
 
 ### Spacing {#spacing}
 
--   Lines should not go further than 80 to 100 characters.
--   In python, indentation blocks use 4 spaces
--   In PHP, indentation blocks use tabs
--   Imports should be ordered alphabetically
--   Separate module imports and `from` imports with a blank line
+- Lines should not go further than 80 to 100 characters.
+- In python, indentation blocks use 4 spaces
+- Imports should be ordered alphabetically
+- Separate module imports and `from` imports with a blank line
 
 Example:
 
-    import argparse
-    import datetime
-    import os
-    import re
-    import shutil
-    import tempfile
+```python
+import argparse
+import datetime
+import os
+import re
+import shutil
+import tempfile
 
-    from StringIO import StringIO
-    from urllib import urlencode
+from StringIO import StringIO
+from urllib import urlencode
+```
 
-### PEP8 {#pep8}
+### Black
 
-When possible, use pep8 to validate your code. Generally, the following
-errors are ignored :
-
-> -   E501 (max 80 chars per line)
+When possible, use black to validate your code.
 
 Example:
 
-    pep8 --ignore=E501 wazo_calld
+```shell
+tox -e black
+```
 
 When possible, avoid using backslashes to separate lines.
 
 Bad Example:
 
-    user = session.query(User).filter(User.firstname == firstname)\
-                              .filter(User.lastname == lastname)\
-                              .filter(User.number == number)\
-                              .all()
+```python
+user = session.query(User).filter(User.firstname == firstname)\
+                          .filter(User.lastname == lastname)\
+                          .filter(User.number == number)\
+                          .all()
+```
 
 Good Example:
 
-    user = (session.query(User).filter(User.firstname == firstname)
-                               .filter(User.lastname == lastname)
-                               .filter(User.number == number)
-                               .all())
+```python
+user = (session.query(User).filter(User.firstname == firstname)
+                           .filter(User.lastname == lastname)
+                           .filter(User.number == number)
+                           .all())
```

Comment:
I'm sure `black` would reformat this though.

Reply:
Yeah I scope myself to styling issue only - otherwise I would not be able to complete this task (now more than one year I'm on it...)
This page should be refactored and some rules are totally useless with black. But not in my scope